operations/commit_queue.go

```diff
@@ -386,33 +378,39 @@ func (p *moduleParams) addModule(ac *legacyClient, rc *legacyClient) error {
 	if err != nil {
 		return err
 	}
-
-	moduleBranch, err := getModuleBranch(p.module, proj)
+	module, err := proj.GetModuleByName(p.module)
 	if err != nil {
-		return errors.Wrapf(err, "could not set specified module: '%s'", p.module)
+		return errors.Wrapf(err, "could not find module '%s'", p.module)
 	}
 
-	commitCount, err := gitCommitCount(moduleBranch, p.ref)
+	commitCount, err := gitCommitCount(module.Branch, p.ref)
 	if err != nil {
 		return errors.Wrap(err, "can't get commit count")
 	}
 	if commitCount == 0 {
 		return errors.New("No commits for module")
 	}
-	if commitCount > 1 && !confirm("Commit queue patch has multiple commits. Continue? (y/n):", false) {
-		return errors.New("patch aborted")
+	if commitCount > 1 && !confirm("Commit queue module patch has multiple commits. Continue? (y/n):", false) {
+		return errors.New("module patch aborted")
 	}
 
-	if p.message == "" {
-		message, err := gitCommitMessages(moduleBranch, p.ref)
-		if err != nil {
-			return errors.Wrap(err, "can't get commit messages")
-		}
+	owner, repo, err := thirdparty.ParseGitUrl(module.Repo)
+	if err != nil {
+		return errors.Wrapf(err, "can't get owner/repo from '%s'", module.Repo)
+	}
 
-		p.message = message
+	patch, err := rc.GetPatch(p.patchID)
+	if err != nil {
+		return errors.Wrapf(err, "can't get patch '%s'", p.patchID)
+	}
+
+	commitMessage, err := gitCommitMessages(module.Branch, p.ref)
+	if err != nil {
+		errors.Wrap(err, "can't get commit messages")
 	}
+	message := fmt.Sprintf("%s %s", strings.TrimSuffix(patch.Description, noCommits), fmt.Sprintf(commitFmtString, commitMessage, owner, repo, module.Branch))
```

Comment:
Can this also be split up for clarity?

Reply:
Also, if you're splitting up commit messages by " <- " should modules not also be split the same way if patch.Description already had commits?

Reply:
done.
I split up the repos with `||` for clarity, so it will look like this:
```
Commit Queue Merge: 'commit1 <- commit2' into 'owner/main_repo:master' || 'module_commit1 <- module_commit2' into 'owner/module1_repo:master' || 'module_commit1 <- module_commit2' into 'owner/module2_repo:master'
```
Is that what you had in mind?